sklearn/tests/test_metaestimators.py

```diff
@@ -69,7 +69,7 @@ def __init__(
     ),
     DelegatorData(
         "SelfTrainingClassifier",
-        lambda est: SelfTrainingClassifier(est),
+        SelfTrainingClassifier,
```

Comment:
some of these are making the code less explicit

Reply:
If `f` is clearly a function, I don't see why `f` would be less explicit than `lambda x: f(x)`. But I can understand you may prefer to add a `lambda ` just to stress that `f` is a function. You could as well add a comment such as `# f is a function`.